package.json

```
@@ -5,9 +5,11 @@
   "scripts": {
     "generate": "node generate-routes.js",
     "postgenerate": "make format",
-    "format": "prettier --write --ignore-path .gitignore ."
+    "format": "prettier --write --ignore-path .gitignore .",
+    "start:fake-seam-connect": "fake-seam-connect --seed"
```

We can keep things simple here and use `start`. Means if we need to start more stuff later we only need to update the script here.

```suggestion
    "start": "fake-seam-connect --seed"
```

---

863b945967a9bda4b6189df79686e6f0f8fc2668

---

https://github.com/seamapi/python-next/pull/67/commits/9d134b3c7794cd497b61619ed1261ca9c72590e4